apisix/cli/ngx_tpl.lua

```diff
@@ -554,6 +554,52 @@ http {
             }
         }
 
+        {% if use_or_1_15 then %}
```

Comment:
do we have to support `openresty 1.15`?
I think we can only support `1.19` and `1.17`. we can update `cli`.

Reply:
what do you think? @spacewander

Reply:
IMHO, 1.15 is released in 2019, and Kong still supports it. Maybe we can give it one more chance.